docs/source/users/getting_started.rst

```diff
@@ -1,5 +1,193 @@
+.. _getting-started:
+
+###############
 Getting Started
-===============
+###############
+
+|Python 2.7| will be needed for running/installing this. Instructions
+for setting up python are available
+`here <https://github.com/mantidproject/fitbenchmarking/wiki/Setting-up-Python>`__.
```

Comment:
This link is redundant, the linked page is now copied below

Reply:
Good spot -- thanks! Fixed